tests/integration_test/services_test.py

```diff
@@ -1293,7 +1296,75 @@ def test_get_data_importer_logs():
                                             {'type': 'zendesk_actions', 'count': 0, 'data': []},
                                             {'type': 'pipedrive_leads_actions', 'count': 0, 'data': []}]
     assert actual['data']["logs"][3]['is_data_uploaded']
-    assert set(actual['data']["logs"][3]['files_received']) == {'rules', 'stories', 'nlu', 'config', 'domain', 'actions'}
+    assert set(actual['data']["logs"][3]['files_received']) == {'rules', 'stories', 'nlu', 'config', 'domain',
+                                                                'actions', 'chat_client_config'}
+
+@responses.activate
+def test_upload_with_chat_client_config():
```

Comment:
upload only chat client config here and assert that event is completed

Reply:
this is because uploading chat client is a non event data and adding test for it will ensure this case